week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,8 +7,20 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
-
+function capitalise(str) {
+  var word = str.split("");
+  const letterUp = word[0].toUpperCase();
+  word.shift();
+  word.unshift(letterUp);
+  return word.join(" ");
+}
+/*function capitalise(str) {
+  word = str.split(" ");
+  for (var i = 0, x = word.length; i < x; i++) {
+    word[i] = word[i][0].toUpperCase() + word[i].substr(1);
+  }
+  return word.join(" ");
```

Comment:
It would be worth you checking whether this line gives you the expected result.

Reply:
i did but because i was trying to try the other function i forget to un comment it